### PR TITLE
IT-16-Elements-Buttons-RightClickMeButton in progress

### PR DESCRIPTION
**Pull request by 1Serhiiqa**
Test for Elements-Buttons-RightClickMeButton in progress

### CODE REVIEW

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs [hunk 1]
```diff
@@ -9,6 +9,11 @@ public class ButtonsPage : BasePage
         private IWebElement DoubleClickMeButton => FindElement(By.CssSelector("button#doubleClickBtn"));
 
         private IWebElement DoubleClickButtonMessage => FindElement(By.CssSelector("p#doubleClickMessage"));
+        private IWebElement RightClickMeButton => FindElement(By.CssSelector("button#rightClickBtn"));
```
Comment (user-yalif) [inline Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs:12]: Please, insert empty space between DoubleClickButtonMessage and RightClickMeButton properties

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs [hunk 2]
```diff
@@ -9,6 +9,11 @@ public class ButtonsPage : BasePage
...
+
+        private IWebElement RightClickMeButtonMessage => FindElement(By.CssSelector("p#rightClickMessage"));
+
```
Comment (user-yalif) [inline Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs:15]: Please, check out that there is only one empty line between properties and methods

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs [hunk 3]
```diff
@@ -29,5 +34,27 @@ public string GetMessageAfterDoubleClick()
...
+            actions.Build()
+                .Perform();
+
+            return this;
+        }
+
+        public string GetMessageAfterRightClickMe()
+        {
+
```
Comment (user-yalif) [inline Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs:54]: Remove empty line

Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs]: done

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs (hunk 1 above)
Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs:13]: done

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs (hunk 2 above)
Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/ButtonsPage.cs:16]: done

**Comment by 1Serhiiqa** [comment]
IT-16 - Yuriy`s comments fixing

**Pull request closed without merging**